app/retail/emails.py

```diff
@@ -551,40 +553,40 @@ def render_new_bounty_roundup(to_email):
 '''
     highlights = [
         {
-            'who': 'octavioamu',
+            'who': 'tcrowe',
             'who_link': True,
-            'what': 'Worked with MARKET Protocol on their calendar navigation.',
-            'link': 'https://gitcoin.co/issue/MARKETProtocol/dApp/295/965',
+            'what': 'Build a feedback form on the Plasma Learning page.',
+            'link': 'https://gitcoin.co/issue/ethsociety/learn-plasma/17/970',
             'link_copy': 'View more',
         },
         {
-            'who': 'srisankethu',
+            'who': 'pinkiebell',
             'who_link': True,
-            'what': 'Completed his first bounty on the Ethereum Utils repo!',
-            'link': 'https://gitcoin.co/issue/ethereum/eth-utils/117/964',
+            'what': 'Built out the Gitcoin Requests highlighted above!',
+            'link': 'https://gitcoin.co/issue/gitcoinco/web/2036/982',
             'link_copy': 'View more',
         },
         {
-            'who': 'tcrowe',
+            'who': 'evgeniuz',
```

Comment:
@mbeacom the svgs we serve dont seem to render in gmail.  any good workarounds youre aware of here? http://bits.owocki.com/a0729b318442/Screen%20Shot%202018-08-24%20at%2010.31.27%20AM.png

Reply:
@owocki Which SVGs seem to be not rendering?

Reply:
look at the screenshot in my comment above or in triage channel

see https://gitcoin.co/_administration/email/roundup if you want to verify that its a SVG specific issue